### PR TITLE
Seo migration

### PR DESCRIPTION
**Pull request by moaiandin**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md

### Summary

SUMMARY_GOES_HERE

### Full changelog

* Updated version.rb to reflect the new directory structure for docs
* Hard coded links updated to not cause 404s
* Mass link update for docs, install and plugins
* Updated menu links
* Introduction of new redirects

### Checklist:
- [X] [Commit message & atomicity](https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [X] Documentation 
- [X] Spellchecked my updates
- [X] Ready to be merged

### CODE REVIEW

File: app/0.10.x/configuration.md [hunk 1]
```diff
@@ -665,7 +665,7 @@ A common pattern is to create a VPN between the two datacenters such that
 the flat network assumption is not violated.
 
 See the clustering reference for more informations:
-https://getkong.org/docs/latest/clustering/
+https://getkong.org/latest/clustering/
```
Comment (coopr) [inline app/0.10.x/configuration.md]: Shouldn't this point to docs.konghq.com/latest/clustering/ ?

File: app/0.11.x/lua-reference/modules/kong.tools.responses.html [hunk 2]
```diff
@@ -182,7 +182,7 @@ <h4><a name="status_codes">status_codes</a></h4>
     Define the most common HTTP status codes for sugar methods.
  Each of those status will generate a helper method (sugar)
  attached to this exported module prefixed with <code>send_</code>.
- Final signature of those methods will be <code>send_&lt;status_code_key&gt;(message, headers)</code>. See <a href="../modules/kong.tools.responses.html#send">send</a> for more details on those parameters.
+ Final signature of those methods will be <code>send_&lt;status_code_key&gt;(message, headers)</code>. See <a href="send">send</a> for more details on those parameters.
```
Comment (coopr) [inline app/0.11.x/lua-reference/modules/kong.tools.responses.html]: Is this missing a # in the anchor tag?

File: app/0.3.x/getting-started/introduction.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (coopr) [inline app/0.3.x/getting-started/introduction.md]: Why is a whole new file appearing? This is curious to me.

**Comment by coopr** [comment]
@moaiandin are you able to respond to the comments? TIA!

File: app/0.4.x/getting-started/introduction.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (travispamaral) [inline app/0.4.x/getting-started/introduction.md]: same as before. Any reason why these intro pages are being treated as new?

Comment (moaiandin) [reply app/0.4.x/getting-started/introduction.md]: Really strange, not sure why its doing that. Same file, nothing changed on it

File: app/0.3.x/getting-started/introduction.md (hunk 3 above)
Comment (moaiandin) [reply app/0.3.x/getting-started/introduction.md]: Nothing changed, not sure why git is being strange here.

File: app/0.11.x/lua-reference/modules/kong.tools.responses.html (hunk 2 above)
Comment (moaiandin) [reply app/0.11.x/lua-reference/modules/kong.tools.responses.html]: Ture, that needs a # indeed.

File: app/0.10.x/configuration.md (hunk 1 above)
Comment (moaiandin) [reply app/0.10.x/configuration.md]: Yes, correct.

File: app/0.11.x/lua-reference/modules/kong.tools.responses.html (hunk 2 above)
Comment (coopr) [reply app/0.11.x/lua-reference/modules/kong.tools.responses.html]: Potentially this issue exists elsewhere - can we presume you'll find and fix all occurrences of this @moaiandin ?

**Comment by moaiandin** [comment]
I've updated the couple of issues and pushed in the changes.

**Pull request merged**